hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/io/compress/bzip2/CBZip2OutputStream.java

```diff
@@ -64,64 +64,64 @@
  * &lt;code&gt;65k + (5 * blocksize)&lt;/code&gt;.
  * </pre>
  *
- * <table width="100%" border="1">
+ * <table border="1">
  * <caption>Memory usage by blocksize</caption>
- * <colgroup> <col width="33%" > <col width="33%" > <col width="33%" >
```

Comment:
I removed/replaced all `attribute`/`tag` that were complained by the `javadoc` and were removed in HTML5, this might slightly impact the final HTML style but I think it should be okay.